tools/marvin/marvin/config/test_data.py

```diff
@@ -419,8 +419,8 @@
         "displaytext": "MySharedOffering",
         "guestiptype": "Shared",
         "supportedservices": "Dhcp,Dns,UserData",
-        "specifyVlan": "True",
-        "specifyIpRanges": "True",
+        "specifyVlan": "False",
```

Comment:
@fmaximus This test data is in use by several test cases, changing the values from 'true' to 'false' may cause regressions. Can you change them back, and in your tests perhaps change the value to False if necessary?

Reply:
This *is* the revert of test data, done in PR #2259

Reply:
@rhtyd this change is exactly the revert you asked for. I think we shd merge in this PR as your comment was legitimate.